### ISSUE TITLE
Model example is not clear on how to load models

### ISSUE THREAD

**Issue by tigerclaw-az** [bellstrand/mongoose-seed-db#1]
I'm receiving an error `error: mongodb connection error Cannot read property 'modelName' of undefined` when trying to use the `loadModels()` against a file based on the Model example.

My model file looks like this:
```
var mongoose = require('mongoose'),
	Schema = mongoose.Schema,
	CardsSchema = new Schema({
		name: String
	}, {
		collection: 'cards'
	});

module.exports = mongoose.model('Cards', CardsSchema);
```

**Comment by bellstrand** [comment]
@ejwaibel I'll take a look at it tonight!

**Comment by bellstrand** [comment]
I only supported "export default" syntax and not "module.exports".
Updated and fixed in 0.0.8

**Comment by tigerclaw-az** [comment]
👍 have confirmed this fixed my issue, thanks!!
